### PR TITLE
adds aphrodisiac nukes

### PR DESCRIPTION
**Pull request by silicons**
you all asked for it

### CODE REVIEW

**Comment by Nukie2557** [comment]
Oh hell yes. Speedmerge this shit.

**Comment by deathride58** [comment]
Do you think god stays in heaven because he, too, fears what he created?

**Comment by LetterN** [comment]
madlad

File: code/modules/antagonists/nukeop/equipment/nuclearbomb.dm [hunk 1]
```diff
@@ -497,6 +497,33 @@
...
 
+//lazy citadel edit begin
+/obj/machinery/nuclearbomb/beer/aphro
+	name = "Nanotrasen-brand nuclear fission explosive"
+	desc = "One of the more successful achievements of the Nanotrasen Corporate Warfare Division, their nuclear fission explosives are renowned for being cheap to produce and devastatingly effective. Signs explain that though this particular device has been decommissioned, every Nanotrasen station is equipped with an equivalent one, just in case. All Captains carefully guard the disk needed to detonate them - at least, the sign says they do. There seems to be a tap on the back. It smells of a sickening sweet flavor."
+
+/obj/machinery/nuclearbomb/beer/aphro/Initialize()
+	. = ..()
+	if(keg)
+		keg.reagents.remove_any(INFINITY)
```
Comment (Iamgoofball) [inline code/modules/antagonists/nukeop/equipment/nuclearbomb.dm:508]: this is not how you do things, you want to go add support for a starting reagent to the /obj/machinery/nuclearbomb/beer path, bro you know this by now

File: code/modules/antagonists/nukeop/equipment/nuclearbomb.dm [hunk 2]
```diff
@@ -497,6 +497,33 @@
...
+		keg.reagents.add_reagent("aphro", 1000)
+
+/obj/machinery/nuclearbomb/beer/aphro/actually_explode()
+	var/turf/bomb_location = get_turf(src)
+	if(!bomb_location)
+		disarm()
+		return
+	if(is_station_level(bomb_location.z))
+		var/datum/round_event_control/E = locate(/datum/round_event_control/vent_clog/aphro) in SSevents.control
```
Comment (Iamgoofball) [inline code/modules/antagonists/nukeop/equipment/nuclearbomb.dm:517]: something tells me this is not the cheapest way to handle this

File: code/modules/events/vent_clog.dm [hunk 3]
```diff
@@ -96,3 +96,29 @@
...
+
+/datum/round_event/vent_clog/aphro
+	reagentsAmount = 100
+	var/reagentid = "aphro"
+	var/annoucemsg = "The scrubbers network is experiencing an unexpected surge of pressurized aphrodisiac. Some ejection of contents may occur."
+
+/datum/round_event/vent_clog/aphro/announce()
+	priority_announce(announcemsg, "Atmospherics alert")
+
+/datum/round_event/vent_clog/aphro/start()
```
Comment (Iamgoofball) [inline code/modules/events/vent_clog.dm:114]: useless override, all you had to do was set randomProbability to 0 and saferChems to list("aphro")

**Comment by Iamgoofball** [comment]
3/10, shoddy codemanship

File: code/modules/antagonists/nukeop/equipment/nuclearbomb.dm (hunk 1 above)
Comment (LetterN) [reply code/modules/antagonists/nukeop/equipment/nuclearbomb.dm:508]: >not even on it's own modular_cit

**Review by Cw3040 — APPROVED** [review]
this is fine

**Comment by jakeramsay007** [comment]
10/10 iamgoofball reviews that aren't just memes, I'm proud of you buddy

**Comment by Iamgoofball** [comment]
@Cw3040 this is not fine

**Pull request closed without merging**